x/mongo/driver/topology/connection_test.go

```diff
@@ -264,8 +264,8 @@ func TestConnection(t *testing.T) {
 					// both of the tests declared below. Both tests also specify a 10ms max connect time to provide
 					// a large buffer for lag and avoid test flakiness.
 
-					{"context timeout is lower", 1 * time.Millisecond, 100 * time.Millisecond, 10 * time.Millisecond},
-					{"connect timeout is lower", 100 * time.Millisecond, 1 * time.Millisecond, 10 * time.Millisecond},
+					{"context timeout is lower", 1 * time.Millisecond, 100 * time.Millisecond, 50 * time.Millisecond},
```

Comment:
Looks like the same change as was done for the [`wait-queue-timeout`](https://github.com/mongodb/specifications/blob/master/source/connection-monitoring-and-pooling/tests/wait-queue-timeout.yml) spec test. Was this erroring with 10ms?

Reply:
Yes, it was intermittently failing while trying to assert that `connect()` returned in under 10ms, so I increased it to 50ms.

Reply:
Sounds good.